.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. You can ensure this by searching the issue list for this
+repository. If there is a duplicate, please close your issue and add a comment
+to the existing issue instead.
+
+If you suspect your issue is a bug, please edit your issue description to
+include the BUG REPORT INFORMATION shown below. If you fail to provide this
+information within 7 days, we cannot debug your issue and will close it. We
+will, however, reopen it if you later provide the information.
+
+---------------------------------------------------
+BUG REPORT INFORMATION
+---------------------------------------------------
+-->
+
+**Description**
+
+<!--
+Briefly describe the problem you are having in a few paragraphs.
+-->
+
```

Comment:
Please also include:

- indication whether this is a problem with the protocol, node or something else
- fields for which OS they are using
- description on how to get the logs for the raiblocks node
- warning not to disclose items for the Raiblocks bug bounty here, they must be communicated via the official channel!